constrained_ik/examples/launch_example.launch

```diff
@@ -2,7 +2,7 @@
     <arg name="use_yaml" />
     <arg name="example" />
```

Comment:
The "example" parameter is used to select the node "type", shouldn't the node "type" be unmutable?